sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/FormRecognizerClient.java

```diff
@@ -71,7 +71,7 @@ public FormTrainingClient getFormTrainingClient() {
      * has failed, or has been cancelled.
      */
     @ServiceMethod(returns = ReturnType.COLLECTION)
-    public SyncPoller<OperationResult, IterableStream<RecognizedForm>>
+    public SyncPoller<OperationResult, List<RecognizedForm>>
```

Comment:
Unclear how Java usually does these docstrings, but I notice in the docstring for the SyncPoller / PollerFlux you don't mention what the poller returns. Python includes this information in the docstring (ex [here](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_form_recognizer_client.py#L96)).

This comment isn't super related to what this PR is trying to do, so feel free to open another PR when / if you want to make this change